src/hotspot/share/opto/mulnode.hpp

```diff
@@ -389,4 +389,12 @@ class MulAddS2INode : public Node {
   virtual uint ideal_reg() const { return Op_RegI; }
 };
 
+//------------------------------CompressBitsNode-------------------------------
+// CompressBits placeholder node
+class CompressBitsNode : public Node {
+public:
+  CompressBitsNode( Node *in1, Node *in2 ) : Node(0,in1,in2) {}
```

Comment:
Style: the spaces are not needed after `"("` and before `")"`. And spaces are needed between the arguments. And `"*"` is better to be next close to the type. So suggest to modify as:
`CompressBitsNode(Node* in1, Node* in2) : Node(0, in1, in2) {}`

Reply:
Thank you. Will fix this.